fixes #3973 - foreman-debug can now upload tarballs

lzap opened the pull request:
I will send an e-mail with instructions how to pick tarballs up. Only us can do
that of course.

lzap committed:
fixes #3973 - foreman-debug can now upload tarballs

domcleal commented on script/foreman-debug:
```diff
@@ -265,3 +265,17 @@ else
 
+echo "You may want to upload the tarball to our public server via rsync. There is a"
+echo "write only directory (readable only by Foreman core developers) for that. Note"
+echo "the transmittion will be UNENCRYPTED using rsync:"
```
transmission

domcleal commented:
Could you also add rsync as a package dependency?  Also to debs (where maybe it should be a recommends or suggestion, can't remember which).

lzap commented:
Fixed, also I added check for rsync. If it is not present, foreman-debug
will not offer upload interactively.

LZ

On Sat, Jan 11, 2014 at 03:42:53AM -0800, Dominic Cleal wrote:

> Could you also add rsync as a package dependency?  Also to debs.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/theforeman/foreman/pull/1132#issuecomment-32093780

## 

Later,

 Lukas "lzap" Zapletal
 irc: lzap #theforeman

skottler commented:
@lzap was there supposed to be a spec change included with that last comment? rsync should be added as a requires in `foreman.spec`.

lzap commented:
You are right, that is better, pushing.

lzap commented:
Debian change will follow shortly.

domcleal commented:
Thanks @lzap, merged as ad07d9a for Foreman 1.4.0.
